app/matplotlib.ipynb

```diff
@@ -233,22 +316,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, (ax1, ax2) = plt.subplots(1, 2)\n",
+    "fig.suptitle(\"subplots\")\n",
+    "ax1.plot(x, y1, color=\"orange\")\n",
+    "ax1.set_title(\"Sin(x)\")\n",
+    "ax1.set_xlabel(\"x\")\n",
+    "ax1.set_ylabel(\"sin(x)\")\n",
+    "\n",
+    "ax2.plot(x, y2, \"tab:green\")\n",
```

Comment:
The color for the cosine plot is specified using `"tab:green"`, which is not a standard color specification in Matplotlib. Consider using a valid color string like `"green"`.